packages/sensors/example/pubspec.yaml

```diff
@@ -12,12 +12,12 @@ dev_dependencies:
     sdk: flutter
   integration_test:
     path: ../../integration_test
-  pedantic: ^1.8.0
+  pedantic: ^1.10.0-nullsafety
 
 flutter:
   uses-material-design: true
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
Does reverting `example` work for this plugin, or are there conflicts?

Reply:
Reverting should work. I forgot that I made this PR a month ago and the example folders were still migrated.